resources/kcp/charts/kyma-environment-broker/templates/environments-cleanup-job.yaml

```diff
@@ -1,47 +1,48 @@
 {{if eq .Values.global.kyma_environment_broker.environmentsCleanup.enabled true}}
+{{- range $index, $job := .Values.setupJobs }}
 apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
-  name: "kcp-kyma-environments-cleanup"
-  namespace: kcp-system
+  name: "{{$job.name}}"
+  namespace: "{{$job.namespace}}"
 spec:
-  schedule: {{.Values.environmentsCleanup.schedule}}
+  schedule: {{$job.schedule}}
   failedJobsHistoryLimit: 5
   successfulJobsHistoryLimit: 1
   concurrencyPolicy: Forbid
   jobTemplate:
     metadata:
       labels:
-        cronjob: environments-cleanup
+        cronjob: {{$job.cronJobName}}
     spec:
       template:
         metadata:
-          name: kyma-environments-cleanup
+          name: {{$job.imageName}}
           labels:
-            cronjob: environments-cleanup
+            cronjob: {{$job.cronJobName}}
           {{- if eq .Values.global.database.embedded.enabled false }}
           annotations:
             sidecar.istio.io/inject: "false"
           {{ end }}
         spec:
-          serviceAccountName: kcp-kyma-environment-broker
+          serviceAccountName: {{$job.name}}
```

Comment:
looks like they previously ran as `KEB`, with this they may need new RBAC as well. Worth checking if they touch kube-api at all.